dash-pipeline/SAI/src/utils.h

```diff
@@ -268,6 +269,10 @@ namespace dash
                     case SAI_IP_ADDR_FAMILY_IPV6:
                         {
                             uint8_t ip[16];
+                            if (std::is_same<T, p4::v1::FieldMatch_LPM>::value) {
+                                // BMv2 cannot support IPv6 LPM with prefix
+                                return SAI_STATUS_NOT_SUPPORTED;
```

Comment:
is this the BMv2's limitation or it is our implementation limit?

also i am a bit hesitated that we should check and fail here, because of 2 reasons:
- It is a utility function and should not have any knowledge on the scenarios
- Also the type check is a bit hacky.

IMO, if this is something we don't support, the change in the template should be good enough.